Trimble.Modus.Components/Hosting/AppBuilderExtensions.cs

```diff
@@ -73,5 +74,32 @@ public static MauiAppBuilder ConfigurePopups(this MauiAppBuilder builder, Action
                 });
             return builder;
         }
+        public static void AllowMultiLineTruncation()
```

Comment:
Can you please move this method as label handler

Reply:
Done